cropper/app/lib/Crops.scala

```diff
@@ -60,7 +60,7 @@ class Crops(config: CropperConfig, store: CropStore, imageOperations: ImageOpera
   def createCrops(sourceFile: File, dimensionList: List[Dimensions], apiImage: SourceImage, crop: Crop, cropType: MimeType)(implicit logMarker: LogMarker): Future[List[Asset]] = {
     logger.info(logMarker, s"creating crops for ${apiImage.id}")
 
-    Future.sequence[Asset, List](dimensionList.map { dimensions =>
```

Comment:
i'm not sure i understand why this type annotation has been removed -- is it just because it's not necessary?

Reply:
yes, it's not necessary and the compiler can infer the correct type. It's motivated by a change in the signature, going from [2 generic arguments in 2.12](https://www.scala-lang.org/api/2.12.20/scala/concurrent/Future$.html#sequence[A,M[X]%3C:TraversableOnce[X]](in:M[scala.concurrent.Future[A]])(implicitcbf:scala.collection.generic.CanBuildFrom[M[scala.concurrent.Future[A]],A,M[A]],implicitexecutor:scala.concurrent.ExecutionContext):scala.concurrent.Future[M[A]]) to [3 in 2.13](https://www.scala-lang.org/api/2.13.15/scala/concurrent/Future$.html#sequence[A,CC[X]%3C:IterableOnce[X],To](in:CC[scala.concurrent.Future[A]])(implicitbf:scala.collection.BuildFrom[CC[scala.concurrent.Future[A]],A,To],implicitexecutor:scala.concurrent.ExecutionContext):scala.concurrent.Future[To])